scripts/bench.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Example: Compile and benchmark the difficulty pallet.
+# ./scripts/bench.sh -p difficulty -cb
+
+PALLET=pallet_creditcoin
+COMMAND=build
+BENCH=1
+BUILD=1
+
+while getopts "fcbp:" opt;do
+    case $opt in
+    (f) COMMAND=check BUILD=0;;
+    (c) BUILD=0;;
+    (p) PALLET="pallet_$OPTARG";;
+    (b) BENCH=0;;
+    esac
+done
+
+if [[ $BUILD -eq 0 ]]
+then
+    cargo $COMMAND --release --features runtime-benchmarks || exit 1;
+fi
+
+if [[ $BENCH -eq 0 ]]
+then
+    ./target/release/creditcoin-node benchmark --chain dev --steps=50 --repeat=30 --pallet $PALLET --extrinsic='*' --execution wasm --wasm-execution=compiled --heap-pages=10000 --output ./runtime/src/weights/$PALLET.rs;
+fi
```

Comment:
@CAGS295 Does it make sense to use this script as the entrypoint for CI instead of the current configuration:
https://github.com/gluwa/creditcoin/blob/dev/.github/workflows/ci.yml#L272 ? I'm not quite sure.

The goal of the CI job it just to make sure benchmarks compile and can be executed without crashing, nothing more.

Reply:
You can do whatever you want with the script, This how I developed the benchmarks and pushed the script for reproducibility. I suggest having one script for compilation and running the benchmarks. This script includes both. I think all the options are necessary, some for statistics properties and the rest to keep the WASM target constant. This is the suggested options to bench the WASM blob runtime. You can set `--pallet` to '*' to benchmark all pallets at once, I couldn't make it work within reasonable time so I left it out.

Reply:
The benchmarks could fail at runtime so I suggest letting it run with --repeat=1 just to make sure it works as expected.